### PR TITLE
Adds where_single_line in configuration documentation

### LINKED ISSUE #2186: `where_single_line` is missing an entry in Configurations.md
This is a documentation issue

### PR DESCRIPTION
**Pull request by iapain**
Fixes #2186

### CODE REVIEW

File: Configurations.md [hunk 1]
```diff
@@ -906,6 +906,37 @@ fn lorem() -> usize {
...
+    Option<T>: Ipsum,
+{
+    ...
+}
+```
+
+#### `true`:
+
+```rust
+impl<T> Lorem for T where
```
Comment (topecongiro) [inline Configurations.md]: The `where` needs to be on the next line. cc [the style guide](https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/items.md#option---where_single_line).

**Comment by topecongiro** [comment]
Thank you for the PR! Just a small typo needs to be fixed.

**Comment by iapain** [comment]
@topecongiro Thanks for review! I've updated PR.

**Comment by topecongiro** [comment]
Thank you!

**Pull request merged**